src/lando/main/scm/hg.py

```diff
@@ -332,7 +332,7 @@ def describe_commit(self, revision_id: str = ".") -> CommitData:
         """Return Commit metadata."""
         return self._describe_revisions(revision_id)[0]
 
-    def describe_local_changes(self) -> list[CommitData]:
+    def describe_local_changes(self, **kwargs) -> list[CommitData]:
```

Comment:
Looks like there is a divergence here in terms of what this method expects and uses in git vs. hg. Can this be made more explicit?